templates/web/base/report/form/user_loggedout.html

```diff
@@ -0,0 +1,52 @@
+[% IF c.cobrand.social_auth_enabled %]
+  <div class="js-new-report-user-hidden">
+  [% IF c.config.FACEBOOK_APP_ID %]
+    <div class="form-box">
+    <button name="facebook_sign_in" id="facebook_sign_in" value="facebook_sign_in" class="btn btn--block btn--social btn--facebook">
+        <img alt="" src="/i/facebook-icon-32.png" width="17" height="32">
+        [% loc('Log in with Facebook') %]
+    </button>
+    </div>
+  [% END %]
+  [% IF c.config.TWITTER_KEY %]
+    <div class="form-box">
+    <button name="twitter_sign_in" id="twitter_sign_in" value="twitter_sign_in" class="btn btn--block btn--social btn--twitter">
+        <img alt="" src="/i/twitter-icon-32.png" width="17" height="32">
+        [% loc('Log in with Twitter') %]
+    </button>
+    </div>
+  [% END %]
+    <div class="form-box hidden-nojs">
+        <button id="js-new-report-user-show" class="btn btn--social btn--social-email">[% loc('Log in with email') %]</button>
+    </div>
+  </div>
+    <button id="js-new-report-user-hide" class="btn btn--provide-update hidden-js js-new-report-user-shown">[% loc('Back') %]</button>
+    <div class="hidden-js js-new-report-user-shown">
+[% END %]
+
+<div id="form_sign_in">
+    <div class="hidden-nojs js-new-report-sign-in-hidden">
+        <p style="display:flex;">
+        <span style="flex: 1 0 0">Tell us about yourself or</span>
+        <a style="flex: 1 0 0" id="js-new-report-show-sign-in">sign in with password to prefill this information</a></span>
+        </p>
```

Comment:
There's no equivalent to this for the password form so once you've clicked this if you realise you actually want to use the email form you have to click "Back" and then "Continue".

Reply:
Yeah, Back there should really be one step back shouldn't it, lazy me.